### DISCUSSION TITLE
Welcome to WineskinServer Discussions!

### DISCUSSION THREAD

**Discussion by Gcenx** [The-Wineskin-Project/WineskinServer#180] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

### Issue section disabled
The issues tab had been disabled as the majority of opened issues failed to fill out the issue template and then some users were getting aggressive when asked to provide the required information.

**Comment by doh-123** [comment]
I find it amazing Wineskin is alive anywhere.  I made Wineskin just as a hobby just to get a few games working several years ago and it ballooned out to something bigger, but I stopped when many things happened in life and I also stopped using Macs at all (I still don't).  Thanks Gcenx for keeping this going... not sure I can be much help, but its cool to see anyways.

**Reply by emendelson** [reply]
Hello @doh-123 ! Wineskin is one of the very best things that ever happened to the Mac. Thank you again for creating it! (You and I were in touch years ago about the last versions that you created.) And, yes, @Gcenx has been doing a spectacular job of keeping it going.

**Comment by Gcenx** [comment]
@doh-123 I’d started to modify then continue Wineskin for the same reason.

In the end it just made more sense to do a tail fork and make alterations needed for more modern versions of macOS/wine. Until upstream wine completed there _long_ overdue changes to officially support a 64Bit only OS the projects mostly makes use of newer WineCX releases.

**Comment by Skriatok** [comment]
Hello, my name is Peter (Skriatok). I love old games (one specifficaly) and I like neat info. Mac user obviously. Im here only because of NWN1 game. Our server is still on an old 32bit NWN1. Enhanced 64bit edition would require a wipe (The death for such an old and small community). Im not a coder, rather a poweruser. @doh-123 made many of us to continue to play without any stupid solutions. Praise @Gcenx for continuing and further work. Everything is going south with 32bit plus M1 and next apple silicons combo. So I found a light at the end of a tunnel. I might be useful for some user experience info and for spreading the word, how beautifuly crazy important this is. I will provide my community with steps of workarounds and yours with Credits. If I make this game working. If not I still love this project. Thank you. Im not planning to spam your discussions and work.
